classifier_TEMPLATE.yaml

```diff
@@ -204,4 +204,15 @@ classify:
     # folder is where classifier output and mp4s will be created.  It is specified relative to the base_data_folder
     classify_folder: "classify"
 
+evaluate:
+    # Evalulates results against pre-tagged ground truth.
+    show_extended_evaluation: False
 
+    # number of seconds between clips required to trigger a a new visit
+    new_visit_threshold: 180
+
+    # false positive's and 'none' can be mapped to the same label as they represent the same idea.
+    null_tags: ["false-positive", "none", "no-tag"]
+
+    #classes
+    classes: ["bird", "possum", "rat", "hedgehog", "stoat", "none"]
```

Comment:
I have a feeling that this should be at the top level as it'll be something than just step should care about. For example, classify and train must still have a list like this hardcoded in them.

It's fine for now but just be aware that it might move.

Reply:
Yep, i think build has a whole bunch of hard coded config